### ISSUE TITLE
Service listens on loopback 127.0.0.1 instead of wild card (0.0.0.0)

### ISSUE THREAD

**Issue by jhawk7** [imker25/samba_exporter#145]
Hi, I cam across this project when searching for a way to get samba metrics into my prometheus instance running on a separate machine. I was looking for a docker option, but I was able to build the project from source and get it running on my target machine (where my samba service runs) but I was unable to hit the metrics endpoint externally. After some digging, I found that "127.0.0.1:9922" is passed as an argument by default in "etc/default/samba_exporter" the for the "samba_exporter.service" exec command (link [here](https://github.com/imker25/samba_exporter/blob/main/install/etc/default/samba_exporter#L6))

Changing it to "0.0.0.0:9922" fixed the issue, but I was wondering why it was setup to use the loopback interface by default. I expected it to listen on all interfaces given the flag value in the parameters.go file [here](https://github.com/imker25/samba_exporter/blob/main/src/tobi.backfrak.de/cmd/samba_exporter/paramters.go#L39). I'd imagine most users would have prometheus running on a separate machine. Is this mentioned in the documentation?

**Comment by imker25** [comment]
Hi,

well that this is the default behavior is documented in the [man page](https://imker25.github.io/samba_exporter/manpages/samba_exporter.1.html)

> The service will start with -web.listen-address=127.0.0.1:9922 by default, in case your prometheus server is running on a different machine you need to change this.

But you are right, since nobody reads the good old man pages anymore I might should add this information to the [service integration page](https://imker25.github.io/samba_exporter/UserDocs/ServiceIntegration/#prometheus) as well, to avoid confusions. I think I will make this more clear in the documentation and close this bug once the updated documentation is published.

And well, the code line you mention is there also by intention. When testing the software it's very useful to have the exporter listen on all interfaces. But for security reasons you might don't want this in a production environment 1).  That's why the `/etc/default/samba_exporter` service configuration file overwrites the default behavior defined in the code.

**1)** samba_exporter exposes data that some might would consider as confidential like user names, share names, ... . So users should be in full control on where it will be exposed. And users should make a consciously decision to do so.

**Comment by jhawk7** [comment]
Makes sense! Thanks for pointing out the man page. I was absolutely checking the other documentation for that info.

**Comment by imker25** [comment]
Fixed with 69da365b25fdf3e278d69d4fe98f565d73b3971f

This ticket will be closed with the next release.

**Comment by imker25** [comment]
Fixed with [V1.36.11](https://github.com/imker25/samba_exporter/releases/tag/1.36.11)
